pkg/metadata/host/host.go

```diff
@@ -145,6 +146,11 @@ func getMeta() *Meta {
 	return m
 }
 
+func getNetworkMeta() *NetworkMeta {
+	nid, _ := util.GetNetworkID()
```

Comment:
We should log the error from `GetNetworkID`. Also in case of error do we want to return a `NetworkMeta` with an empty string or return `nil` ?

Reply:
`getMeta` follows agent5 payload which use to send an empty JSON dict. We could do better here since network ID is new. It depends on the backend implementation.

Reply:
I change this to return nil if the network meta is missing